utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -334,17 +334,18 @@ pub enum Command {
 	///     different state transition function.
 	///
 	/// To make testing slightly more dynamic, you can disable the state root  check by enabling
-	/// `ExecuteBlockCmd::no_check`. If you get signature verification errors, you should
-	/// manually tweak your local runtime's spec version to fix this.
+	/// `ExecuteBlockCmd::no_check`. If you get signature verification errors, you should manually
+	/// tweak your local runtime's spec version to fix this.
 	///
 	/// A subtle detail of execute block is that if you want to execute block 100 of a live chain
 	/// again, you need to scrape the state of block 99. This is already done automatically if you
 	/// use [`State::Live`], and the parent hash of the target block is used to scrape the state.
 	/// If [`State::Snap`] is being used, then this needs to be manually taken into consideration.
 	///
-	/// This executes the same runtime api as normal block import, namely `Core_execute_block`. If
-	/// `ExecuteBlockCmd::no_check` is set, it uses a custom, try-runtime-only runtime
-	/// api called `TryRuntime_execute_block_no_check`.
+	/// This does not executes the same runtime api as normal block import, namely
+	/// `Core_execute_block`. Instead, it uses `TryRuntime_execute_block`, which can optionally
```

Comment:
Maybe we can provide doc-links to these?

Reply:
As noted, these are string literals that are used to call into runtime APIs, nothing to link to: https://github.com/paritytech/substrate/blob/42450b524c7f69d0bba7562aee4d8e9c4ee8f714/utils/frame/try-runtime/cli/src/commands/execute_block.rs#L197